docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -147,6 +153,47 @@ The API ML TLS requires servers to provide HTTPS ports. Each of the API ML servi
     - Authentication is service-dependent
     - Recommended to use the Authentication and Authorization Service for authentication
 
+
+#### Authentication endpoints
+
+The API Gateway contains two REST API authentication endpoints: `auth/login` and `auth/query`.
+
+The `/login` endpoint allows to authenticate mainframe user credentials and returns an authentication token. The login request requires the user credentials in one of the following formats:
+  * Basic access authentication
+  * JSON with user credentials 
+
+When successfully authenticated, the response to the request is an empty body and a token in a secure `HttpOnly` cookie named `apimlAuthenticationToken`, otherwise it returns a 401 status code.
```

Comment:
I would chop this sentence. It's kind of big, and has a lot of information. 
So upon successful authentication, the response is an empty body. That's clear. Now what happens to the token (in a secure `httponly` cookie that is called `apimlAuthenticationToken`)? Will a user get this token upon successful authentication, or a different token (also called differently)? or does a user just get a 401 rc?

Reply:
The user will get a 401 error without any token if the authentication fails

Reply:
I changed it to:
When successfully authenticated, the response to the request is an empty body and a token in a secure `HttpOnly` cookie named `apimlAuthenticationToken`. In case of authentication failure, the response is a 401 status code.

Reply:
ps: 
1. When successfully authenticated, the response to the request is an empty body and a token in a secure `HttpOnly` cookie named `apimlAuthenticationToken`. 
2.  When authentication failed, a user gets 401 status code.
I re orgamnized it like that, as you suggested in the comment below